### PR TITLE
Got the framework for the testing done

### PR DESCRIPTION
**Pull request by Ciscoslahs**
still trying to understand the fields I need to use to test.

### CODE REVIEW

**Review by bleuflame47 — COMMENTED** [review]
This looks good

**Review by bleuflame47 — APPROVED** [review]
This is great scaffolding to use continue forward.

**Review by Stowne1 — COMMENTED** [review]
looks good to me.

**Review by Stowne1 — APPROVED** [review]
looks good pt 2

**Comment by Stowne1** [comment]
Helped review and test this part.

File: DataProject/TestDatabaseWarehouse.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Class TestDatabaseWarehouse(unittest.TestCase):
+
+        """
+        -assertRaises and assertEquals could be good too for DatabaseWarehouse _init_
+        -assertEquals for DatabaseWarehouse item_selected (testing line that it moves to)
+        -assertIsInstance for DatabaseWarehouse run
+        -assertIsInstance for Operations _init_
+        """
+        """Have no clue what to do for __init__ method"""
```
Comment (sgeinitz) [inline DataProject/TestDatabaseWarehouse.py:13]: No need to have an __init__ method for a unittest test class. If you want, you can have a method called, setUp(), though, which can be used to create some class attributes/fields that can be used within the unit test methods (e.g. test_run).

**Pull request merged**